website.js

```diff
@@ -184,6 +190,9 @@ app.get('/user/:id', function(req, res) {
         }
       }
       request(info, function (error, response, body) {
+        var prePic = JSON.parse(body).logo
```

Comment:
You should cache the `JSON.parse` call, because it's quite an expensive task.

```javascript
var json = JSON.parse(body); // cache the output of the `JSON.parse` call
var prePic = json.logo;
var name = json.name;

// re-use here
```